### PR TITLE
[1LP][RFR]adding test_no_power_controls_on_archived_vm

### PR DESCRIPTION
**Pull request by lkhomenk**
__Adding test__  to check Power Toolbar on Archived VM

{{pytest: cfme/tests/infrastructure/test_vm_power_control.py -k "test_no_power_controls_on_archived_vm" --use-provider vsphere65-nested --long-running}}

### CODE REVIEW

File: cfme/common/vm.py [hunk 1]
```diff
@@ -584,7 +587,8 @@ def _looking_for_state_change():
                 self.load_details(refresh=True)
                 return self.get_detail(properties=detail_t) == desired_state
             else:
-                return 'currentstate-' + desired_state in self.find_quadicon().state
+                return 'currentstate-' + desired_state in self.find_quadicon(
```
Comment (quarckster) [inline cfme/common/vm.py]: Use `format()` please.

Comment (lkhomenk) [inline cfme/common/vm.py]: @quarckster done

File: cfme/common/vm.py [hunk 2]
```diff
@@ -294,7 +294,9 @@ def find_quadicon(
             try:
                 if not search.has_quick_search_box():
                     # TODO rework search for archived/orphaned VMs
-                    if self.is_vm:
+                    if from_any_provider:
+                        navigate_to(self, 'All')
+                    elif self.is_vm:
                         navigate_to(self, 'AllForProvider', use_resetter=False)
```
Comment (izapolsk) [inline cfme/common/vm.py]: I guess you can merge this elif with else. they both do the same

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 3]
```diff
@@ -314,6 +322,19 @@ def test_no_template_power_control(provider, soft_assert):
...
+    """ Ensures that no power button is displayed from details view of archived vm
+
+    Prerequisities:
+        * Archived VM
+    Steps:
+        * Open the view of VM Details
+        * Verify the Power toolbar button is not visible
+    """
+    testing_vm.load_details()
+    soft_assert(not toolbar.exists("Power"), "Power displayed in template details!")
```
Comment (izapolsk) [inline cfme/tests/infrastructure/test_vm_power_control.py]: please use regular assert instead

**Pull request merged**